rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    person.add_rental(self)
+    book.add_rental(self)
```

Comment:
- [ ] the rental should add itself to the book and the person by being pushed to book.rentals and person.rentals not by calling the method.